build/parse.y

```diff
@@ -531,6 +531,22 @@ ident:
 	{
 		$$ = &LiteralExpr{Start: $1, Token: $<tok>1}
 	}
+| '(' idents comma_opt ')'
+  {
+		if len($2) == 1 && $3.Line == 0 {
```

Comment:
what is going on with the indents on this line versus after the else?

Reply:
please always use gofmt

Reply:
It's mixed tabs and spaces (and they're mixed somewhere else in this file). What should be used here?

Reply:
Replaced with tabs as almost elsewhere in this file.